quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -96,15 +95,22 @@ class
   --
   -- @
   --   data Action RegState a where
-  --     Spawn      ::                           Action RegState ThreadId
-  --     Register   :: String -> Var ThreadId -> Action RegState (Either ErrorCall ())
-  --     KillThread :: Var ThreadId           -> Action RegState ()
+  --     Spawn      ::                           Action RegState () ThreadId
```

Comment:
This documentation change is not valid anymore if I am not mistaken

Reply:
I've fixed this now.